pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> A tool to manage network booting of machines.
+> More information: <https://github.com/danderson/netboot/tree/master/pixiecore>.
+
+- Boot with `netboot.xyz` on the same network:
+
+`pixiecore quick xyz --dhcp-no-bind`
```

Comment:
I'm not sure about the `quick xyz`. I assume `quick` is an argument and `xyz` a variable, but it also seems to be a file extention.

Reply:
They are both arguments, `xyz` is a reference to the [netboot.xyz](https://netboot.xyz/) website.

Reply:
Ah, now I get it. I assume `quick` is optional or can be replaced with something else. Can you also replace `xyz` with something else? If not, just apply the first change.
```suggestion
- Boot with `netboot.xyz` on the same network:

`pixiecore {{quick}} xyz --dhcp-no-bind`
```
(The `{{ }}` tells the client to hightlight that part. We do this for everything you have to put something in).

Also, I `netboot.xyz` looks like a file. I think that might be confusing.

Reply:
Yes, `xyz` could be replaced by `ubuntu`, or `fedora` for instance. Let me apply the suggestion.